packages/fx-core/templates/plugins/resource/frontend/bicep/frontendHosting.bicep

```diff
@@ -0,0 +1,22 @@
+@minLength(3)
```

Comment:
should we consider the bicep file as static file?
currently our static files are stored centralized under fx-core/templates directory since it can simplify webpack work.

Reply:
Yes. The file is placed under *fx-core/templates* directory.